rules/macos/execution_scripting_osascript_exec_followed_by_netcon.toml

```diff
@@ -0,0 +1,66 @@
+[metadata]
+creation_date = "2020/12/07"
+maturity = "production"
+updated_date = "2020/12/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects AppleScript execution via the Apple Scripting Interpreter (osascript) followed by a network connection from
+the same process within a short time period. Adversaries may abuse AppleScript for execution and command and
+control.
```

Comment:
```suggestion
Detects execution via the Apple script interpreter (osascript) followed by a network connection from
the same process within a short time period. Adversaries may use malicious scripts for execution and command and
control.
```

Reply:
since in can be used for JavaScript and AppleScript, decided to remove specific AppleScript reference and leave it open to either.  another option could be specify both.